definitions/divorce/json/CaseEventToFields/CaseEventToFields-nonprod.json

```diff
@@ -430,5 +430,96 @@
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "amendPetitionForRefusalSol",
+    "CaseFieldID": "labelAmendPetitionForRefusalSol",
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "READONLY",
+    "PageID": "amendpetitionstart",
+    "PageLabel": "Submit amended application",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "ShowSummaryChangeOption": "No"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "amendPetitionForRefusalSol",
+    "CaseFieldID": "labelAmendPetitionForRefusalSolPara-1",
+    "PageFieldDisplayOrder": 2,
+    "DisplayContext": "READONLY",
+    "PageID": "amendpetitionstart",
+    "PageLabel": "Submit amended application",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "ShowSummaryChangeOption": "No"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "amendPetitionForRefusalSol",
+    "CaseFieldID": "labelAmendPetitionForRefusalSolPara-2",
+    "PageFieldDisplayOrder": 3,
+    "DisplayContext": "READONLY",
+    "PageID": "amendpetitionstart",
+    "PageLabel": "Submit amended application",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "ShowSummaryChangeOption": "No"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "amendPetitionForRefusalSol",
+    "CaseFieldID": "labelAmendPetitionForRefusalSolPara-3",
+    "PageFieldDisplayOrder": 4,
+    "DisplayContext": "READONLY",
+    "PageID": "amendpetitionstart",
+    "PageLabel": "Submit amended application",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "ShowSummaryChangeOption": "No"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "amendPetitionForRefusalSol",
+    "CaseFieldID": "labelAmendPetitionForRefusalSolPara-4",
+    "PageFieldDisplayOrder": 5,
+    "DisplayContext": "READONLY",
+    "PageID": "amendpetitionstart",
+    "PageLabel": "Submit amended application",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "ShowSummaryChangeOption": "No"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "amendPetitionForRefusalSol",
+    "CaseFieldID": "labelAmendPetitionForRefusalSolPara-5",
+    "PageFieldDisplayOrder": 6,
+    "DisplayContext": "READONLY",
+    "PageID": "amendpetitionstart",
+    "PageLabel": "Submit amended application",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "ShowSummaryChangeOption": "No"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "amendPetitionForRefusalSol",
+    "CaseFieldID": "labelAmendPetitionForRefusalSolPara-6",
+    "PageFieldDisplayOrder": 7,
+    "DisplayContext": "READONLY",
+    "PageID": "amendpetitionstart",
+    "PageLabel": "Submit amended application",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "ShowSummaryChangeOption": "No"
```

Comment:
Adding new labels. Content signed off can be found [here](https://docs.google.com/document/d/1U5E_DhAX1FdcIg5ukZRCcZ2XVPwTKBeYbLf4Lm9T3_k/edit?usp=sharing) and also in the Jira ticket.

Reply:
Removed the PageLabel property, page already displaying exact same label from the event definition